src/mscorlib/src/System/Math.CoreCLR.cs

```diff
@@ -27,16 +27,28 @@ public static partial class Math
 
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern double Acos(double d);
-        
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        public static extern double Acosh(double d);
```

Comment:
(nit) - I know there are existing inconsistencies here, but it would be nice to have a consistent `value` vs `d` vs `a` naming.

Reply:
I would prefer `value`, however, I went with the overriding rule that new APIs should be consistent with existing APIs in the same class where we have `13` using `d`, `5` using `value`, and `3`, using `a`.

Updating the argument names would be a breaking change.

Reply:
> Updating the argument names would be a breaking change.

😞 Well I guess this is the best we can do given the circumstances.